app/api/organizations/api.rb

```diff
@@ -0,0 +1,21 @@
+module Organizations
+  class API < Grape::API
+    version 'v1', using: :path
+    format :json
+
+    include Authentication::Helpers
+
+    namespace :organizations do
+      desc 'auto-complete organization names'
+      params do
+        requires :term, type: String, desc: 'Term to match.'
```

Comment:
are we indexing `lat`, `lon` and locations too? i was wondering if it makes sense to add those to our query too. That can totally be separate PR if indexing by location is needed.

Reply:
We only index city and country names as I don't believe we have an actual `lat` and `long` for these Organizations, maybe @jonallured can clarify if we do. We could add filters for country and city very easily to this request, as additional params which we send to ES. I'm not sure there's any need for that right now, but definitely an enhancement.